contracts/Witnet.sol

```diff
@@ -1,10 +1,15 @@
-pragma solidity ^0.5.0;
+pragma solidity >=0.5.3 <0.7.0;
 pragma experimental ABIEncoderV2;
 
 import "./BufferLib.sol";
 import "./CBOR.sol";
 
 
+/**
+ * @title A library for decoding Witnet request results
```

Comment:
```suggestion
 * @title A library for decoding Witnet request results.
```

Reply:
I left titles without points on purpose... don't know if it's the best or not... :frowning_man: